Domains/4-Application/Connectors/CivilInfrastructureFramework/CifGeotechnical.ecschema.xml

```diff
@@ -0,0 +1,167 @@
+        <BaseClass>cifcmn:CivilPresentation</BaseClass>
+        <ECProperty propertyName="DepthInterval_TopDepth" typeName="double" displayLabel="Top Depth" category="Depth_Interval_Depth_Interval" priority="300000"/>
+        <ECProperty propertyName="DepthInterval_BottomDepth" typeName="double" displayLabel="Bottom Depth" category="Depth_Interval_Depth_Interval" priority="299999"/>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="ElementOwnsComponentStationPropertiesClassAspect" modifier="None" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="bis:GeometricElement3d"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="ComponentStationPropertiesClassAspect"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECRelationshipClass typeName="ElementOwnsDepthInformationAspect" modifier="None" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="bis:GeometricElement3d"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="DepthInformationAspect"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECRelationshipClass typeName="ElementOwnsDepthIntervalAspect" modifier="None" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="bis:GeometricElement3d"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="DepthIntervalAspect"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="LocationAspect">
+        <BaseClass>cifcmn:CivilPresentation</BaseClass>
+        <ECProperty propertyName="Location_Name" typeName="string" displayLabel="Name" category="Location_Location" priority="300000"/>
+        <ECProperty propertyName="Location_Depth" typeName="double" displayLabel="Depth" category="Location_Location" priority="299999"/>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="ElementOwnsLocationAspect" modifier="None" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="true">
+            <Class class="bis:GeometricElement3d"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="LocationAspect"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="LocationOrientation" displayLabel="Orientation">
+        <BaseClass>bis:ElementUniqueAspect</BaseClass>
```

Comment:
aspect classes should end in 'Aspect'.

https://www.itwinjs.org/bis/guide/naming-guidelines/rules-and-recommendations/#class

Reply:
Appended 'Aspect' to class name and implement mapping in Civil Connector